cmd/ls.go

```diff
@@ -30,6 +31,7 @@ var lsCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 		spaceDID, err := did.Parse(cmd.Flags().Arg(0))
 		if err != nil {
+			cmd.SilenceUsage = false
```

Comment:
Here, and many other places: Why do this? The default value of an unset bool is false. Unless we have overridden it somewhere else?

Reply:
+1 not sure why it's needed to set this over and over